internal/testrunner/runners/system/runner.go

```diff
@@ -238,10 +239,22 @@ func (r *runner) run() error {
 }
 
 func (r *runner) tearDown() {
+	if logger.IsDebugMode() {
+		// Sleep to give some time for humans to scrutinize the current
```

Comment:
I disabled it on purpose in one of recent PR not to slow down testing. Especially if we add next tests in the future. If you prefer we can leave it here or add a command argument not to wipe out the cluster state.

Reply:
Yeah, it is only adding the sleep in debug mode. I wonder if we can add a check here to only do this if the test fails? If the test passes it's unlikely that we want to inspect the state of the system, right?